### DISCUSSION TITLE
lotus daemon

### DISCUSSION THREAD

**Discussion by chukangkang** [filecoin-project/lotus#9261] [Build and Sync]
[lotus.log](https://github.com/filecoin-project/lotus/files/9485852/lotus.log)

 lotus sync wait
Worker: 3; Base: 0; Target: 2134418 (diff: 2134418)
State: header sync; Current Epoch: 2133913; Todo: 505
Validated 0 messages (0 per second)

It's been in sync for half a month, and it's still in sync. Genesis is in sync
lotus version 1.17.0+mainnet+git.2830429ad
No public IP address

**Comment by chukangkang** [comment]
The situation is similar to #8001, but there is no solution

**Comment by LexLuthr** [comment]
@chukangkang Are you planning to run a full node for historical data queries? Otherwise, there is no need to sync from genesis. 
https://lotus.filecoin.io/lotus/manage/chain-management/#lightweight-snapshot

Even if you are planning to run a full node, I would recommend going with a full node snapshot.
